jumpscale/packages/owncloud/package.py

```diff
@@ -0,0 +1,25 @@
+import toml
```

Comment:
I think we can directly use `jumspcale.data.serializers.toml` and `jumpscale.sals.fs` here instead?

Reply:
done for slas.fs, but for `toml` the `load` and `dump` functions are not exposed in `jumpscale.data.serializers.toml`, only `loads` and `dumps` , so I'm sticking with `toml` original import for now.